applications/minotari_console_wallet/src/wallet_modes.rs

```diff
@@ -255,13 +255,13 @@ fn wallet_or_exit(
             },
             _ => {
                 info!(target: LOG_TARGET, "Starting TUI.");
-                tui_mode(handle, config, base_node_config, wallet)
+                tui_mode(handle, config, base_node_config, wallet).await
             },
         }
     }
 }
 
-pub fn tui_mode(
+pub async fn tui_mode(
```

Comment:
Where did the need for these to become async come in?

Reply:
all the services now require calls to the key manager to start. That is an async call.